mmseg/apis/ote/apis/segmentation/inference_task.py

```diff
@@ -50,6 +51,7 @@
                                                            set_hyperparams)
 from mmseg.apis.ote.apis.segmentation.configuration import OTESegmentationConfig
 from mmseg.apis.ote.apis.segmentation.ote_utils import InferenceProgressCallback
+from mmseg.core.evaluation.metrics import f_score
```

Comment:
Do we have a plan to replace mDice to f_score?

Reply:
it depends on you